File: app/src/main/java/com/ipd/taxiu/ui/activity/message/MessageDetailActivity.java
package com.ipd.taxiu.ui.activity.message;

import android.os.Bundle;

import com.ipd.taxiu.R;
import com.ipd.taxiu.ui.BaseUIActivity;

import org.jetbrains.annotations.NotNull;
import org.jetbrains.annotations.Nullable;

/**
 * Created by Miss on 2018/7/31
 * 详情
 */
public class MessageDetailActivity extends BaseUIActivity{
    @Override
    protected int getContentLayout() {
        return R.layout.activity_message_detail;
    }

    @Override
    protected void initView(@Nullable Bundle bundle) {
        initToolbar();
    }

    @Override
    protected void loadData() {

    }

    @Override
    protected void initListener() {

    }

    @NotNull
    @Override
    protected String getToolbarTitle() {
        return "详情";
    }
}

File: app/src/main/java/com/ipd/taxiu/ui/activity/message/MessageDetailActivity.kt
package com.ipd.taxiu.ui.activity.message

import android.app.Activity
import android.content.Intent
import android.os.Bundle
import com.ipd.taxiu.R
import com.ipd.taxiu.ui.BaseUIActivity

/**
Created by Miss on 2018/8/13
详情
 */
class MessageDetailActivity : BaseUIActivity() {
    companion object {
        fun launch(activity: Activity){
            val intent = Intent(activity,MessageDetailActivity::class.java)
            activity.startActivity(intent)
        }
    }

    override fun getContentLayout(): Int = R.layout.activity_message_detail
    override fun getToolbarTitle(): String = "详情"

    override fun initView(bundle: Bundle?) {
        initToolbar()
    }

    override fun loadData() {
    }

    override fun initListener() {
    }
}